lambdas/reconcile-migrated-vcs/src/main/java/uk/gov/di/ipv/core/reconcilemigratedvcs/ReconcileMigratedVcsHandler.java

```diff
@@ -103,7 +103,7 @@ public ReconciliationReport handleRequest(Request request, Context context) {
         var forkJoinPool =
                 forkJoinPoolFactory.getForkJoinPool(
                         request.parallelism() == null
-                                ? DEFAULT_PARALLELISM
+                                ? Runtime.getRuntime().availableProcessors()
```

Comment:
Do you know if this will do what you want in the lambda runtime? I didn't get useful results with it when I tried in the migration script.

It's also not necessarily a good default - I would expect most of the time in the script is waiting for requests to EVCS/DynamoDB, so won't be utilizing a processor.

Anyway, you've played with the parallelism so you can tell me what's most effective!

Reply:
I wanted to work out how many CPUs were available when upping the memory to max. 6 it turns out. There is only a small increase in perf (and probably not statistically significant with a sample size of 1) between 4 threads and 6. But seeing as we have those CPUs available, and we also have the ability to set the parallelism through the request to whatever we want, I figured may as well leave it in.